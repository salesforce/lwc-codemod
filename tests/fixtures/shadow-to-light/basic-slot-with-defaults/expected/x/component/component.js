import { LightningElement } from 'lwc'

export default class extends LightningElement {
  static renderMode = "light";
  onScroll() {
    console.log('scroll!')
  }
  renderedCallback() {
    console.log('slot', this.querySelector('.slot-wrapper'))
  }
}
