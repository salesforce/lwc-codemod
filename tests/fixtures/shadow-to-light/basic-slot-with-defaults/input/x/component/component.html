<template>
    <slot name="baz" onscroll={onScroll}>
        <div>default!</div>
    </slot>
</template>
