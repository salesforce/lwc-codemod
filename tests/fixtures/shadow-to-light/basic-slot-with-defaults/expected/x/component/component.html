<template lwc:render-mode="light">
    <div onscroll={onScroll} class="lwc-slot-wrapper"><slot name="baz">
        <div>default!</div>
    </slot></div>
</template>
